### ISSUE TITLE
Enabling and Disabling Firebase Performance Monitoring

### ISSUE THREAD

**Issue by CostasAthan** [TobiasBuchholz/Plugin.Firebase#333]
1. Is [Firebase Performance Monitoring](https://firebase.google.com/docs/perf-mon) part of [Plugin.Firebase.Analytics](https://www.nuget.org/packages/Plugin.Firebase.Analytics/) or [Plugin.Firebase](https://www.nuget.org/packages/Plugin.Firebase/)?
2. If yes, is there a way to [enable or disable performance monitoring](https://firebase.google.com/docs/perf-mon/disable-sdk) through the plugin?

**Comment by AdamEssenmacher** [comment]
No, not currently. For that to work, we'd need an updated Firebase.PerformanceMonitoring binding package for iOS. This would require a community contribution or sponsorship over at https://github.com/AdamEssenmacher/GoogleApisForiOSComponents
